setup.py

```diff
@@ -115,25 +115,42 @@ def lua_libs(package='luajit'):
 
 basedir = os.path.abspath(os.path.dirname(__file__))
 
+def get_lua_build_from_arguments():
+    lua_l = get_option('--lua-l')
+    lua_I = get_option('--lua-I')
```

Comment:
These are difficult to distinguish in some fonts. Let's make the names explicit.
```python
    lua_lib = get_option('--lua-lib')
    lua_includes = get_option('--lua-includes')
```

Reply:
Resolved in 32f0d3c